### ISSUE TITLE
DateTime fields for Toml Frontmatter should export as DateTimes, not Strings

### ISSUE THREAD

**Issue by xelivous** [sveltia/sveltia-cms#233]
Followup to #228
It seems like datetime widgets are serializing to string format instead of raw datetimes. They can still be interpreted by most frontends in the "incorrect" format but it should still likely be fixed to follow [the spec](https://toml.io/en/v1.0.0#offset-date-time).

What it exports as right now:
```md
+++
date = "2024-10-24T00:00:00"
+++
```

What it should be:
```md
+++
date = 2024-10-24T00:00:00
+++
```

**Comment by kyoshino** [comment]
I forgot to mention this in #228. I know TOML has date format without quotes, but as per the [DateTime widget spec](https://decapcms.org/docs/widgets/#datetime), the data type is a string. I’ll check how Netlify/Decap CMS handles it. If there is incompatibility, I’ll fix it.

**Comment by kyoshino** [comment]
Just tested with Decap CMS and it also quoted dates when saving an entry as TOML:

```toml
date = "2024-10-25T06:55:00.000Z"
```

So this would be wontfix.

**Comment by kyoshino** [comment]
I’ve found a related Netlify/Decap CMS and developed a solution for this:

- https://github.com/decaporg/decap-cms/issues/3583

**Comment by kyoshino** [comment]
The native date/time format supported landed in [v0.43.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.43.0). Please reload the CMS to see the change.
